bluebrain/repo-bluebrain/packages/neurodamus/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from llnl.util import tty
+
+from spack.package import *
+
+from .neurodamus_model import NeurodamusModel, copy_all, make_link
+
+
+class Neurodamus(NeurodamusModel):
+    """The next-generation AllInOne Neurodamus deployment.
+
+    Neurodamus includes BBP's simulation suite with all internal sim models.
+    """
+
+    homepage = "https://bbpgitlab.epfl.ch/hpc/sim/neurodamus-models"
+    git = "ssh://git@bbpgitlab.epfl.ch/hpc/sim/neurodamus-models.git"
+
+    version("develop", submodules=True)
+    # Let the version scheme be different to avoid users loading it by mistake and getting puzzled
+    version("2023.04", branch="bump/2023-april", submodules=True)
+
+    depends_on("py-neurodamus", type=("build", "run"))
+
+    phases = ["build_model", "merge_hoc_mod", "build", "install"]
```

Comment:
Could `build_model` and `merge_hoc_mod` not be phases? It seems that the former could be folded into `build`, and the other one could just be `@before("build")`...

Reply:
@matz-e yeah I think we could do that. This is mostly for visibility during the recipe execution, because some of these phases are long and would give us the ability to see where it failed, in case.

Reply:
I reworked that a bit because `merge_hoc_mod` is indeed an aux step. In turn `build_model` is a full build without neurodamus mods, and therefore was kept but renamed to `build_model_only`